vmm/src/config.rs

```diff
@@ -755,6 +766,56 @@ impl DiskConfig {
             .map_err(Error::ParseDisk)?
             .unwrap_or(Toggle(false))
             .0;
+        let bw_size = parser
+            .convert("bw_size")
+            .map_err(Error::ParseDisk)?
+            .unwrap_or_default();
+        let bw_one_time_burst = parser
+            .convert("bw_one_time_burst")
+            .map_err(Error::ParseDisk)?
+            .unwrap_or_default();
+        let bw_refill_time = parser
+            .convert("bw_refill_time")
+            .map_err(Error::ParseDisk)?
+            .unwrap_or_default();
+        let ops_size = parser
+            .convert("ops_size")
+            .map_err(Error::ParseDisk)?
+            .unwrap_or_default();
+        let ops_one_time_burst = parser
+            .convert("ops_one_time_burst")
+            .map_err(Error::ParseDisk)?
+            .unwrap_or_default();
+        let ops_refill_time = parser
+            .convert("ops_refill_time")
+            .map_err(Error::ParseDisk)?
+            .unwrap_or_default();
+        let bw_tb_config = if (bw_size | bw_refill_time) != 0 {
+            Some(TokenBucketConfig {
+                size: bw_size,
+                one_time_burst: Some(bw_one_time_burst),
+                refill_time: bw_refill_time,
+            })
+        } else {
+            None
+        };
+        let ops_tb_config = if (ops_size | ops_refill_time) != 0 {
```

Comment:
What happens if `ops_size > 0` but `ops_refill_time == 0` sounds like you really need both to be set for something to be usable.

Reply:
That's right. I will need to revisit this part when I add the HTTP API support. I want to make sure the CLI and HTTP API handles them in the same way.

In the end, the "TokenBucket::new()" will add limiting only when both "size" and "refill_time" are non-zero.

Reply:
I don't think you need a HTTP API to change the rate limit. It should be fixed as device creation time.

Reply:
Given the HTTP API will require to have both `size` and `refill_time` for a `TokenBucket`. I updated the parsing on CLI to do the same thing.